docs/manifests.md

```diff
@@ -20,6 +20,8 @@ The use of Manifest Deployer is quite similar to the use the `kubectl apply` com
 
 ## Example
 
+**Note:** The file type must be `.yaml` not `.yml`.
```

Comment:
Hi @pepsi1k, thanks for the contribution!

We should have this documented that we only match the .yaml and not .yml files, but I think this shouldn't be in the Example section. Can you move it to the Overview category? Maybe something like:

Maybe something like:

By default, k0s reads all manifest files under /var/lib/k0s/manifests that end with .yaml and ensures that their state matches the cluster state. It does not read files with other extensions, such as .yml. Moreover, on removal of a manifest file, k0s will automatically prune all of its associated resources.